qiskit/pulse/commands/command.py

```diff
@@ -15,25 +15,35 @@
 """
 Base command.
 """
+import re
+
 from abc import ABCMeta, abstractmethod
 
 from qiskit.pulse.exceptions import PulseError
 
 from .instruction import Instruction
 
 
-class Command(metaclass=ABCMeta):
+class MetaCount(ABCMeta):
+    """Meta class to count class instances."""
+    def __new__(mcs, name, bases, namespace, **kwargs):
```

Comment:
I think the issue may be the extra **kwargs?

Reply:
Could be. Let me push again and see

Reply:
If I remove the `**kwargs` I get a pylint warning on my local machine.

Reply:
I think the problem is likely that you are inheriting from `ABCmeta` and instead could inherit directly from `type`?